spec/features/self_service/firms_index_spec.rb

```diff
@@ -68,6 +80,11 @@ def when_i_am_on_the_principals_firms_index_page
     firms_index_page.load
     expect(firms_index_page).to be_displayed
   end
+  alias_method :and_i_am_on_the_principals_firms_index_page, :when_i_am_on_the_principals_firms_index_page
```

Comment:
(wow amazing, would have expected Rubocop to complain about no whitespace before the alias. #school_day)